index.html

```diff
@@ -6,7 +6,411 @@
+              консультанты подберут для вас наиболее выгодные<br>
+              условия кредита. Выгодные для банка, разумеется.
+            </p>
+          </li>
+        </ul>
+      </section>
+
+      <aside class="rare_order">
+        <h2 class="rare_order-title">
+          Доставим редкий заказ под заказ
+        </h2>
+        <a class="rare_order-info" href="#">
+          <span class="visually-hidden">
+            Информация
+          </span>
+        </a>
+      </aside>
+
+      <section class="about">
+        <h2 class="about-title">О нас</h2>
+        <p class="about-description">
+          Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.
+        </p>
+        <p class="about-description">
+          Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:
+        </p>
+        <ul class="about-list">
+          <li class="about-item">
+            <b class="company">
+              Меловые линии
+            </b>
+          </li>
+          <li class="about-item">
+            <b class="company">
+              СДИК
+            </b>
+          </li>
+          <li class="about-item">
+            <b class="company">
+              ТоксБерри
+            </b>
+          </li>
+        </ul>
+        <a class="about-link" href="#">Подробнее о нас</a>
+      </section>
+
+      <section class="contacts">
+        <h2 class="contacts-title">Контакты</h2>
+        <p class="contacts-description">
+          Вы можете забрать товар сами, заехав в наш офис.<br>
+          Заодно, сможете проверить работоспособность покупки.
+        </p>
+        <h3 class="contacts-subtitle">Интернет-магазин "Девайс"</h3>
+        <address class="contacts-address">
+          Санкт-Петербург, набережная реки Карповки, 5, литера П.<br>
+          <u class="contacts-nubmer">
+            <a class="contacts-number-link" href="tel:+78128121212">Тел.: +7 (812) 812-12-12</a>
+          </u>
+        </address>
+        <h3 class="contacts-subtitle">Время работы</h3>
+        <ul class="contacts-shedule">
+          <li class="contacts-shedule-item">
+            пн—пт: с 10:00 до 22:00
+          </li>
+          <li class="contacts-shedule-item">
+            сб—вс: с 10:00 до 19:00
+          </li>
+        </ul>
+        <a class="contacts-link" href="#">Напишите нам</a>
+      </section>
+
+      <section class="subscribe">
+        <h2 class="subscribe-title">
+          Подпишитесь на рассылку
+        </h2>
+        <p class="subscribe-description">
+          Еженедельный дайджест с обзорами новинок,
+          длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.
+        </p>
+        <form class="subscribe-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="subscribe-email">
+            <span class="visually-hidden">
+              e-mail
+            </span>
+          </label>
```

Comment:
Нет смысла плодить сущности
```suggestion
          <label for="subscribe-email" class="visually-hidden">
              E-mail
          </label>
```

Reply:
А ещё можно засунуть лейблы в уже существующий текс

```html
<h2 class="subscribe-title">
  <label for="subscribe-email">Подпишитесь на рассылку</label>
</h2>
```